src/hooks/useStripeTerminal.tsx

```diff
@@ -93,6 +96,8 @@ export function useStripeTerminal(props?: Props) {
     setUserCallbacks,
   } = useContext(StripeTerminalContext);
 
+  const _isInitialized = useCallback(() => isInitialized, [isInitialized]);
```

Comment:
@arekkubaczkowski -- instead of using this memoized value for `isInitialized`, can we use the underlying `isInitialized` value returned from the context? I believe that memoizing the boolean leads to incorrect "First initialize the Stripe Terminal SDK before performing any action" errors being raised to the user.

See https://github.com/stripe/stripe-terminal-react-native/issues/381#issuecomment-1261056698.

Reply:
Tying up this loose end -- I think we probably want our `initialize` function to wait until our `isInitialized` state has been set to `true` before returning. I have not put together a patch to test this yet.